lib/db/leveldb_dbinstance.go

```diff
@@ -83,6 +84,31 @@ func newDBInstance(db *leveldb.DB, location string) *Instance {
 	return i
 }
 
+// Update does transitions to the current db version if necessary
+func (db *Instance) Update(cfgVersion int) {
```

Comment:
I'd like something a little bit more descriptive here as Update can mean many things for a database. `UpdateSchema`? `UpdateFormat`?